sdks/go/pkg/beam/artifact/materialize.go

```diff
@@ -172,7 +236,7 @@ func writeChunks(stream jobpb.ArtifactRetrievalService_GetArtifactClient, w io.W
 	return nil
 }
 
-func legacyMaterialize(ctx context.Context, endpoint string, rt string, dest string) ([]*jobpb.ArtifactMetadata, error) {
+func legacyMaterialize(ctx context.Context, endpoint string, rt string, dest string) ([]*pipepb.ArtifactInformation, error) {
```

Comment:
Do you know what the state is of getting rid of the legacy protocol altogether?

Reply:
I think the artifact server for Dataflow still uses legacy protocol.